_includes/macros/faqs.html

```diff
@@ -0,0 +1,67 @@
+{# ==========================================================================
+
+   faqs()
+
+   ==========================================================================
+
+   Description:
+
+   Builds FAQ markup when given:
+
+   faqs                      : An array of FAQs.
+
+   options (optional)        : An object used to customize
+                               the FAQ markup.
+
+   options.title             : Title to use for FAQ markup.
+
+   options.additional_classes: Extra classes added to the FAQ markup.
+
+   ========================================================================== #}
+{% macro render(faqs, options={}) -%}
+    {% set additional_classes = options.additional_classes  %}
+    {% set title = options.title  %}
+    <section class="{{ additional_classes }}">
+        {% if title %}
+            <div class="u-js-only">
+                <h1 class="h2">
+                    {{ title }}
+                </h1>
+                <p class="short-desc">
+                    If you don’t see your question below, contact us at
+                    <a href="mailto:Invitations2cfpb@consumerfinance.gov">
```

Comment:
Will this email always be the same for every FAQ? Should this be an available option?